frontend/src/views/SecretOverviewPage/SecretOverviewPage.tsx

```diff
@@ -111,9 +111,18 @@ export const SecretOverviewPage = () => {
     try {
       // create folder if not existing
       if (secretPath !== "/") {
+        // /hello/world -> ["", "hello","world"]
         const path = secretPath.split("/");
-        const directory = path.slice(0, -1).join("/");
-        const folderName = path.at(-1);
+        // if its empty string on join use and OR gate to convert to /
+        // /hello
+        const directory =
```

Comment:
Is directly kind of like a parent path or the folder right above the current one the user is in? From first glace, i couldn't tell what it was